modules/runner/src/integrationTest/java/org/apache/ignite/internal/calcite/ItDataTypesTest.java

```diff
@@ -22,20 +22,18 @@
 import java.util.List;
 import java.util.Set;
 import java.util.stream.Collectors;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 /**
  * Test SQL data types.
  */
-@Disabled("https://issues.apache.org/jira/browse/IGNITE-15107")
 public class ItDataTypesTest extends AbstractBasicIntegrationTest {
     /**
      * Before all.
      */
     @Test
     public void testUnicodeStrings() {
-        sql("CREATE TABLE string_table(key int primary key, val varchar)");
+        sql("CREATE TABLE string_table(key int primary key, val varchar) with partitions=10,replicas=2");
```

Comment:
perhaps, it would be better to change default in `TableConfigurationSchema` and file a ticket to revert

Reply:
don`n like this idea, i preferred to store it as is.

Reply:
done ! good catch !